### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by luigimolinaro** [AdaptiveScale/lxdui#34]
I'm having this problem after a new fresh installation : 

```
[2018-03-09 15:44:05,554] ERROR in app: Exception on / [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/lxdui/app.py", line 202, in home
    return render_template("login.html")
  File "/usr/local/lib/python2.7/dist-packages/flask/templating.py", line 132, in render_template
    ctx.app.update_template_context(context)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 764, in update_template_context
    context.update(func())
  File "/usr/local/lib/python2.7/dist-packages/lxdui/app.py", line 110, in get_lxc_version
    ver = float((output_rez).strip())
ValueError: invalid literal for float(): 2.0.11
127.0.0.1 - - [09/Mar/2018 15:44:05] "GET / HTTP/1.1" 500 -

```

**Comment by vhajdari** [comment]
Hi @luigimolinaro, are you using LXDUI 1.0 with a snap install of LXD?  

If so, keep in mind that version 1.0 doesn't work with the LXD snap because the connection to the LXD daemon is done via unix socket.  The LXD team has deprecated the PPA repos, so if you want to use 1.0 with LXD you'll have to install LXD and LXC from the official backports in the Ubuntu archive.  More information is available here: https://insights.ubuntu.com/2017/11/20/lxd-weekly-status-24-lxd-2-20
If you are on Xenial you can install it via `apt install -t xenial-backports lxd lxd-client`

LXDUI 2.0 is under active development to fully support the latest snap version of LXD and once it is released 1.0 will be deprecated as well.  I'd recommend waiting for the next release.

If someone contributes a fix for this particular issue I'll review it and prioritize it accordingly.

**Comment by vhajdari** [comment]
Version 2.0-beta of LXDUI has been released.  Please use the new version.
